java-agent/src/nativeMain/kotlin/com/epam/drill/agent/configuration/AgentArguments.kt

```diff
@@ -99,9 +99,9 @@ data class AgentArguments(
             value = packagePrefixes,
             description = "Configure package prefixes for scanning and instrumentation",
         ),
-        AgentArguments::coverageRetentionLimitBytes.name to AgentParameter(
-            type = coverageRetentionLimitBytes.toType(),
-            value = coverageRetentionLimitBytes.toString(),
+        AgentArguments::coverageRetentionLimit.name to AgentParameter(
+            type = coverageRetentionLimit.toType(),
+            value = coverageRetentionLimit,
```

Comment:
1. Update description:
`Coverage retention queue size limit. Defaults to `*default value*`. Accepts human-readable data size format, e.g.: "512Mb", "1Gb"`

2. Does parser allow for frictional values, e.g. `"0.5 Gb"`? Check and update the description accordingly.

Reply:
done